### PR TITLE
Created header, which hides on scroll

### PR DESCRIPTION
**Pull request by trChris**
Created header, which hides on scroll and appears when user stop scrolling (in 2 seconds).

### CODE REVIEW

File: src/app/shared/header/header.component.ts [hunk 1]
```diff
@@ -1,81 +1,41 @@
...
+      header.classList.remove('hide');
+    }); // always show header after reloading page on any scroll height
+
+    const header = document.getElementById('header');
+    let isScrolling: number;
+
+    window.addEventListener('scroll', (event) => {
+      header.classList.add('hide');
+      header.classList.remove('notransition');
+      window.clearTimeout(isScrolling);
```
Comment (dmitryy90) [inline src/app/shared/header/header.component.ts]: please use observable with debaunce

File: src/app/shared/header/header.component.ts [hunk 2]
```diff
@@ -1,81 +1,42 @@
...
-      filter(direction => direction === Direction.Up)
-    );
-
-    const goingDown$ = scroll$.pipe(
-      filter(direction => direction === Direction.Down)
-    );
-
-    goingUp$.subscribe(() => (this.isVisible = true));
-    goingDown$.subscribe(() => (this.isVisible = false));
+    const header = document.getElementById('header');
```
Comment (dmitryy90) [inline src/app/shared/header/header.component.ts]: we dont use jquery way

File: src/app/shared/header/header.component.ts [hunk 3]
```diff
@@ -1,81 +1,42 @@
...
+
+    const isScrolling = fromEvent(window, 'scroll')
+      .subscribe(() => {
+        let doc = document.documentElement;
```
Comment (dmitryy90) [inline src/app/shared/header/header.component.ts]: cant get the point. why do we need this?
it looks you do some obsolete actions

File: src/app/shared/header/header.component.ts [hunk 4]
```diff
@@ -1,81 +1,42 @@
...
+        } else {
+          header.classList.add('hide');
+          header.classList.remove('notransition'); // run when user is scrolling
+        }
+      });
+
+    const stoppedScrolling = fromEvent(window, 'scroll').pipe(
+      debounce(() => interval(2000))
+    ).subscribe(() => {
+      header.classList.remove('hide');
```
Comment (dmitryy90) [inline src/app/shared/header/header.component.ts]: we have such directive as ngClass, please use it

Comment (trChris) [reply src/app/shared/header/header.component.ts]: Done.

**Pull request merged**